src/samples/audio/detect_test.c

```diff
@@ -48,6 +48,21 @@
 /* default number of samples before detection is activated  */
 #define KEYPHRASE_DEFAULT_PREAMBLE_LENGTH 0
 
+/* 990 ms of data considering frame rate 16KHz
+ * 990 ms = amount of data micro speech nn operates on
+ */
+#define KWD_NN_KEY_LEN         15840
+
+#define KWD_NN_NUM_OF_CHANNELS 1
+
+/* ~2 seconds of samples
+ * acquisition buffer maximum size
+ * used as input for detection algorithms
+ * it contains the necessary amount of
+ * data required by a specific algorithm
+ */
+#define KWD_NN_IN_BUFF_SIZE    (2 * KWD_NN_KEY_LEN * KWD_NN_NUM_OF_CHANNELS)
```

Comment:
Can't the 2 be changed to a macro that says it's the size of a sample? Or what does the 2 mean?